packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -57,10 +68,13 @@ BuildApp() {
 
   # Default value of assets_path is flutter_assets
   local assets_path="flutter_assets"
-  # The value of assets_path can set by add FLTAssetsPath to AppFrameworkInfo.plist
-  FLTAssetsPath=$(/usr/libexec/PlistBuddy -c "Print :FLTAssetsPath" "${derived_dir}/AppFrameworkInfo.plist" 2>/dev/null)
-  if [[ -n "$FLTAssetsPath" ]]; then
-    assets_path="${FLTAssetsPath}"
+  # The value of assets_path can set by add FLTAssetsPath to
+  # AppFrameworkInfo.plist.
+  if PlistHasField "FLTAssetsPath" "${derived_dir}/AppFrameworkInfo.plist"; then
+    FLTAssetsPath=$(/usr/libexec/PlistBuddy -c "Print :FLTAssetsPath" "${derived_dir}/AppFrameworkInfo.plist" 2>/dev/null)
+    if [[ -n "$FLTAssetsPath" ]]; then
+      assets_path="${FLTAssetsPath}"
```

Comment:
This setting was never tested and regressed sometime in the past few months. Moving the asset override into Dart will let us cover it with a unit test at least.

Something here: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/build_system/targets/ios.dart#L286

Reply:
I support that.  In the meantime this just added error checking to the script whole script.  Check out the delta now, hardly touched it.